### ISSUE TITLE
Not Found: /api/v4/templates/menu_main/

### ISSUE THREAD

**Issue by feralfer** [MicroPyramid/Django-CRM#459]
Hie, when running python manage.py runserver
I get 

System check identified no issues (0 silenced).
January 26, 2022 - 23:11:18
Django version 3.2, using settings 'crm.settings'
Starting development server at http://127.0.0.1:8000/
Quit the server with CONTROL-C.
**Not Found: /api/v4/templates/menu_main/
[26/Jan/2022 23:11:22] "GET /api/v4/templates/menu_main/ HTTP/1.1" 404 7957**

Any idea how to fix that, please ? Cheers

**Comment by PoSihDun** [comment]
Yeah fam you have two options I believe. Make a menu_main.html file, or just download a different version. I have had best success with 0.6 thus far.
